### PR TITLE
Tools.json sorting pt. 1

### PR DESCRIPTION
**Pull request by RarkGrames**
#### Summary
```SUMMARY: Infrastructure "Sorted out tools.json into several category files"```

#### Purpose of change
Sorting out tools.json into separate category files in json\items\tool for easier handling.

#### Describe the solution
Doing bathroom.json and cooking.json

#### Testing
Game loads properly. Double checked every item manually to be sure.

#### Additional context
Broke up the whole 7000 line file into 8 separate PRs for easier review.

### CODE REVIEW

File: data/json/items/tool/cooking.json [hunk 1]
```diff
@@ -35,5 +561,400 @@
...
+    "symbol": ")",
+    "color": "light_gray",
+    "looks_like": "pot_canning",
+    "container_data": { "contains": "2 L", "seals": true, "watertight": true },
+    "qualities": [ [ "COOK", 3 ], [ "BOIL", 2 ], [ "CONTAIN", 1 ], [ "CHEM", 1 ] ],
+    "use_action": "HEAT_FOOD",
+    "flags": [ "ALLOWS_REMOTE_USE" ]
+  },
+  {
+    "id": "rock_pot",
```
Comment (I-am-Erk) [inline data/json/items/tool/cooking.json]: This one probably should go into obsolete.json

**Comment by RarkGrames** [comment]
Ok, I changed both.

File: data/json/items/tools.json [hunk 2]
```diff
@@ -5648,21 +4788,6 @@
     "color": "brown",
     "use_action": "LADDER"
   },
-  {
-    "id": "still",
```
Comment (ZhilkinSerg) [inline data/json/items/tools.json]: Should be returned back. It was just deleted.

**Comment by cainiaowu** [comment]
Great, after breaking up item jsons, please also check the recipes, those files are also notorious long and messy.

**Pull request merged**